sklearn/decomposition/_sparse_pca.py

```diff
@@ -342,6 +342,9 @@ class MiniBatchSparsePCA(_BaseSparsePCA):
     the data.  The amount of sparseness is controllable by the coefficient
     of the L1 penalty, given by the parameter alpha.
 
+    For an example of usage see:
+    - :ref:`sphx_glr_auto_examples_decomposition_plot_faces_decomposition.py`
```

Comment:
This example is a good comparison between methods, not a usage example.

Reply:
That's right. I can change "example of usage" to "example comparing sparse PCA to PCA"